### DISCUSSION TITLE
Visualizing the South Sudan 2008 Census by State, Gender, and Age Category with Plotnine

### DISCUSSION THREAD

**Discussion by tongakuot** [has2k1/plotnine#825] [2024 Plotnine Contest]
### Authors

Alier Reng

### Links

https://github.com/tongakuot/dataviz-plotnine/blob/main/barplots.ipynb

### Full description

In this comprehensive tutorial, I will demonstrate how to visualize the South Sudan 2008 Census data using the `plotnine` library in Python. This tutorial will guide you through the following steps:

1) __Creating Bar Graphs__: Learn how to plot basic bar graphs to represent the census data effectively.
2) __Using Facet Grid__: Discover the `facet_grid` functionality in plotnine to create multiple subplots, allowing for a more granular and insightful view of the data across different states, genders, and age categories.
3) __Customizing Plots__: Hone your skills in customizing bar graphs by adjusting aesthetics, themes, labels, and other graphical parameters to make your visualizations more informative and visually appealing.

By the end of this tutorial, you will have a solid understanding of how to use `plotnine` for data visualization, particularly in the context of categorical data analysis. This tutorial is designed for both beginners and intermediate users of plotnine who are looking to enhance their data visualization skills.

### Code repository

https://github.com/tongakuot/dataviz-plotnine/blob/main/barplots.ipynb
![cover-img-02](https://github.com/has2k1/plotnine/assets/77814470/7f3913de-ec0b-49c7-b437-2686a0e27817)

**Comment by has2k1** [comment]
![bp](https://github.com/user-attachments/assets/39c23b9d-567c-447e-9c79-116cab4c9b43)
<hr/>

![bp2](https://github.com/user-attachments/assets/2c5b65c5-506a-49e0-9d82-793851b28193)
<hr/>

![bp3](https://github.com/user-attachments/assets/e20e7bc1-fc9b-4f69-bd00-2742780038ee)

**Comment by has2k1** [comment]
@tongakuot, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.

**Reply by tongakuot** [reply]
Great, thank you! This was an exciting exercise. Much appreciated.
